feeds/process.go

```diff
@@ -200,7 +203,21 @@ func (f *Feed) ProcessFeed() {
 				}
 				if cfg.Filters[i].FilterRegex.MatchString(item.Title) {
 					contentTruncated := false
-					notification := cfg.Repo + " was tagged: " + item.Title + "\nLink: " + item.Link
+					var changeType UpdateType
+					var notification string
+
+					// we check here if last tag hasnt changed
```

Comment:
I think it should be possible to actually distinguish 'new tag', 'retag' and 'description change'. It happens not very often, but 'retag' I think should have it's own notification (process where the commit have changed, but tag name is the same)